### PR TITLE
add Ole Streicher to JOSS paper author list (arbitrary author order for now) [skip ci]

### PR DESCRIPTION
**Pull request by slayoo**
@olebole, please confirm.
We are inviting all contributors and packagers to join the author list (but the process has just started)

### CODE REVIEW

File: paper/paper.md [hunk 1]
```diff
@@ -23,6 +23,9 @@ authors:
   - name: James Tappin
     orcid: 
     affiliation: "6"
+  - name: Ole Streicher
+    orcid:
```
Comment (olebole) [inline paper/paper.md]: My OrCID id is 0000-0001-7751-1843
```suggestion
    orcid: 0000-0001-7751-1843
```

File: paper/paper.md [hunk 2]
```diff
@@ -38,6 +41,7 @@ affiliations:
    index: 5
  - name:  2 RAL Space, STFC Rutherford Appleton Laboratory, UK 
    index: 6
+ - name: Leibniz-Institut für Astrophysik Potsdam (AIP), Potsdam, Germany
```
Comment (olebole) [inline paper/paper.md:44]: The affiliation is correct, but the index: 7 is missing.
```suggestion
 - name: Leibniz-Institut für Astrophysik Potsdam (AIP), Potsdam, Germany 
   index: 7
```

**Review by olebole — COMMENTED** [review]
@slayoo, thank you for the invitation! I am happy to be added to the author list!

**Comment by slayoo** [comment]
thank you Ole!

**Pull request merged**